content/en/dynamic_instrumentation/_index.md

```diff
@@ -34,20 +34,21 @@ further_reading:
 
 Dynamic instrumentation allows you to add instrumentation into your running production systems without any restarts and at any location in your application's code, including third-party libraries. You can add or modify telemetry for logs, metrics, spans, and corresponding tagging, from the Datadog UI. Dynamic Instrumentation has low overhead and has no side effects on your system.
 
-If you are interested in trying out the latest user experience improvements for Dynamic Instrumentation, consider opting into the [Symbol Database private beta][17].
+If you are interested in trying out the latest user experience improvements for Dynamic Instrumentation, consider opting into the [autocomplete and search public beta][17].
 
 ## Getting started
 
 ### Prerequisites
 
 Dynamic Instrumentation requires the following:
 
-- [Datadog Agent][1] 7.44.0 or higher is installed alongside your service.
+- [Datadog Agent][1] 7.44.0 (recommended 7.45.0) or higher is installed alongside your service.
```

Comment:
whats the reason for this recommendation?

if we recommend 7.45. lets put it as requirement  - if its not worse than 7.44 lets keep 7.44 and not do this

Reply:
We need 7.45.0 for SymDB. It's the version that adds the symdb reverse proxy.

Reply:
@EWachnowezki we are not talking about the SymDB only here but DI in general. So we are indicating what is the minimum to have DI working. if you want SymDB working on top of it then you follow the recommendation to a newer version

Reply:
I tend to say that while it is slightly confusing, we should only list the recommended version as the minimum version in the docs site. I think customers will have a much better first impression of the product if SymDB is enabled, so to my mind it is worth it even if it adds some onboarding friction. I don't think we should change the onboarding page minimum or the backend enablement - folks who are using v7.44 should still be able to use it.

Reply:
+1. lets upgrade to v7.45 here. both version are a year old - so if people upgrade it would be v7.54...